content/chainguard/chainguard-images/how-to-use/use-chainguard-helm-charts/index.md

```diff
@@ -66,66 +85,70 @@ global:
 
 ### Deploy a Chainguard Helm chart using a Kubernetes pull secret
 
-1. Authenticate with chainctl and generate a pull token
+To begin, authenticate with chainctl and generate a pull token.
 
-    ```sh
-    chainctl auth login
-    chainctl auth configure-docker --pull-token --save --ttl=24h
-    ```
+```sh
+chainctl auth login
+chainctl auth configure-docker --pull-token --save --ttl=24h
+```
 
-    This creates a short-lived pull token (24 hours). Change the `ttl=` switch to adjust how long your token will exist.
+This token expires in 24 hours by default, which can be modified using the
+`--ttl` flag. It sets the duration for the validity of the token. The maximum
+valid value is `8760h` (equivalent to 365 days), Valid unit strings range from
+nanoseconds to hours and are `ns`, `us`, `ms`, `s`, `m`, and `h`, for example
+`--ttl=24h`.
 
-2. Find the username and password that are contained in the pull token, as in this sample output:
+Find the username and password that are contained in the pull token, as in this sample output:
 
-    ```sh
-    chainctl auth configure-docker --pull-token --save --ttl=24h
-                                        
-      ✔ Selected folder chainguard.edu.
+```sh
+chainctl auth configure-docker --pull-token --save --ttl=24h
+                                    
+  ✔ Selected folder chainguard.edu.
 
-    To use this pull token in another environment, run this command:
+To use this pull token in another environment, run this command:
 
-        docker login "cgr.dev" --username "45a0c61ea6fd977f050c5fb9ac06a69eed764595/095b0c7ea9d68679" --password "eyJhbGciOiJSUzI1NiJ9.eyJhdWQiOiJodHRwczovL2lzc3Vlci5lbmZvcmNlLmRldiIsImV4cCI6MTc0OTczODQ2NSwiaWF0IjoxNzQ5NjUyMDY2LCJpc3MiOiJodHRwczovL3B1bGx0b2tlbi5pc3N1ZXIuY2hhaW5ndWFyZC5kZXYiLCJzdWIiOiJwdWxsLXRva2VuLTAxY2MwODkwYzA5N2ZmMzk1MDUyMWY4NWFmYmEyZDUwMGM0ODQxOWEifQ.ET7ywPUkMk5wN6p0INqhNtdnOVELySqdjp-qWedVmJkLrWlZhdFodU43P4uuR-LJ3Z9mVmd9fjDWpBtZnsCFHbczkENPzOiAFP9fsJhO_2dXT3rXCPK84ddJgRLe6oDlMA3VSa0XEclfTyBcaG4RlrgkVaGhtS7gone4Egff7bKX5Y6-TUxxLiVvCA_l_YmOixUss_Mj1Qxxb81sCeh7x4FSpOGWtmU2Z7Hy6B_rGk17zXMO_GYcuyzAMxfFdQl1Ov18t7KxymQwIoS7UF1fx_5ECR8fgArLM8NikGOjzkiQZuSzeI_hl_GnUFdPTAAhmjpJEWO0isiSPWgpkUPx5scoSUm6jzfduvRgGcmjRxT_pq6MWzFJNw9gv9gVehJuW5lKzNIgMTfJXO5Roba8WCwwxiUknhZXP8DeD_kdAN2-JbkfOYg3aPVU5jFTtA6TJKlh0uQA5OGN5hG_PnyzIr0vu4VVninJTWm66RppdlffhG-1xY9lpXgD2k2TIhygFL8iEBNszq0siLVA3uTH6NZY8iGRFqziUAGnyD80aHn52tIeCBBAOyS6qfcRLzqO6dQX95uscdCOuy-5rxU9n4208m5duLXdZtVWa9gp2vg-OmxnCPVdXmPCTA6RF43gDVkxKGMfvkUkTW1nKNvIUx_ikC9tLHDuZdi8FKLeYEg"
+    docker login "cgr.dev" --username "45a0c61ea6fd977f050c5fb9ac06a69eed764595/095b0c7ea9d68679" --password "eyJhbGciOiJSUzI1NiJ9.eyJhdWQ... # Token truncated"
```

Comment:
```suggestion
docker login "cgr.dev" --username "45a0c61ea6fd977f050c5fb9ac06a69eed764595/095b0c7ea9d68679" --password "eyJhbGciOiJSUzI1NiJ9.eyJhdWQ... # Token truncated"
```

Reply:
That particular indent is in the code being quoted